core/src/main/kotlin/org/evomaster/core/search/StructuralElement.kt

```diff
@@ -132,6 +137,7 @@ abstract class StructuralElement (
 
     fun addChildToGroup(child: StructuralElement, groupId: String){
         verifyChildrenToInsert(child)
+        preChildrenSetup(listOf(child))
```

Comment:
shouldn't this be done after checking validity of inputs?

Reply:
This was changed by @ZhangMan1126